packages/frontend/src/pages/SqlRunner.tsx

```diff
@@ -96,6 +98,32 @@ const SqlRunnerPage = () => {
         sqlQueryMutation,
     });
 
+    const maxLimit = useMemo(
+        () => health.data?.query.maxLimit || 5000,
+        [health],
+    );
+    const showMissingLimitWarning = useMemo(() => {
+        if (sql) {
+            const selectStatements = sql.match(/select\s+/gi);
+            const limitClauses = sql.match(/limit\s+/gi);
```

Comment:
you could have `limit` as a column or alias (or for this regex even stuff like *un*limited) , which will break this regex, but I guess that's ok.